FireRender.Maya.Src/FireRenderViewport.cpp

```diff
@@ -672,52 +690,39 @@ void FireRenderViewport::resizeFrameBufferStandard(unsigned int width, unsigned
 {
 	// Update the RPR context dimensions.
 	m_contextPtr->resize(width, height, false);
-	m_contextPtr->TryCreateDenoiserImageFilters();
 
 	// Resize the pixel buffer that
 	// will receive frame buffer data.
-	m_pixels.resize(width * height);
+	m_texture.Resize(width, height);
 
-	// Perform an initial frame buffer read and update the texture.
-	readFrameBuffer();
-	updateTexture(m_pixels.data(), width, height);
+	if (IsDenoiserUpscalerEnabled())
+	{
+		m_textureUpscaled.Resize(width * 2, height * 2);
```

Comment:
Is 2 a constant that we never expect to be changed?

Reply:
Yes, because we use upscaler 2x mode - "RIF_AI_UPSCALE_MODE_BEST_2X"